config/betterquesting/DefaultQuests/Quests/HowtoGeneratePow-AAAAAAAAAAAAAAAAAAAACQ==/HarnessingChaos-_O6stWM3RR-qJdcvs0i-Mw==.json

```diff
@@ -0,0 +1,108 @@
+{
+  "questIDHigh:4": -221049436640951009,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 4053311655056132369,
+      "questIDLow:4": -5840241591017037228
+    }
+  },
+  "questIDLow:4": -6186301913252184525,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "gregtech:gt.blockmachines",
+        "Count:3": 1,
+        "Damage:2": 32028,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Harnessing Chaos",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "You’ve mastered antimatter production, but raw antimatter alone won\u0027t run your machines. To turn that potential into unparalleled power, it’s time to [warn]harness the chaos[/warn] itself!\n\n[quest]Introducing the Shielded Lagrangian Annihilation Matrix![/quest]\n\nThe SLAM creates raw energy by reacting the Semi-Stable Antimatter with normal matter in a controlled environment.\n\nEvery cycle, the generator will react all antimatter and matter in hatches instantly, converting it all into energy directly. The EU output is based on the equal amounts of antimatter and matter annihilated, therefore, imbalance between the amounts will waste your precious energy.\n\nDepending on the matter used to annihilate, the antimatter amount gets an exponent applied to it, directly increasing the amount of energy produced from 1L of antimatter - this means the more you annihilate in one go, the more efficient it is!\n\nThe maximum EU for a cycle is limited by the capacity of all laser source hatches on the structure (Limited to 64), even on wireless mode.\n\n[note]It can\u0027t output more than 9.22e18 EU in a cycle, or just below 43 16M UXV Laser Sources (If you\u0027re crazy enough for it)\n[/note]\n\nThe Formula and the exponents for each of the possible matters are:\n\nGenerated EU \u003d (§3Antimatter§r^§6exponent§r) * 1.000.000.000\n\nMolten Copper - §61.00§r\nMolten UIV SC Base - §61.02§r\nMolten UMV SC Base - §61.03§r\nMolten Black Dwarf Matter - §61.04§r"
```

Comment:
```suggestion
      "desc:8": "You’ve mastered antimatter production, but raw antimatter alone won\u0027t run your machines. To turn that potential into unparalleled power, it’s time to [warn]harness the chaos[/warn] itself!\n\n[quest]Introducing the Shielded Lagrangian Annihilation Matrix![/quest]\n\nThe SLAM creates raw energy by reacting the Semi-Stable Antimatter with normal matter in a controlled environment.\n\nEvery cycle, the generator will react all antimatter and matter in hatches instantly, converting it all into energy directly. The EU output is based on the equal amounts of antimatter and matter annihilated. Therefore, imbalance between the amounts will waste your precious energy.\n\nDepending on the matter used to annihilate, the antimatter amount gets an exponent applied to it, directly increasing the amount of energy produced from 1L of antimatter - this means the more you annihilate in one go, the more efficient it is!\n\nThe maximum EU for a cycle is limited by the capacity of all laser source hatches on the structure (limited to 64), even on wireless mode.\n\n[note]It can\u0027t output more than 9.22e18 EU in a cycle, or just below 43 16M UXV Laser Sources (If you\u0027re crazy enough for it)\n[/note]\n\nThe Formula and the exponents for each of the possible matters are:\n\nGenerated EU \u003d (§3Antimatter§r^§6exponent§r) * 1.000.000.000\n\nMolten Copper - §61.00§r\nMolten UIV SC Base - §61.02§r\nMolten UMV SC Base - §61.03§r\nMolten Black Dwarf Matter - §61.04§r"
```